composer.json

```diff
@@ -57,6 +57,9 @@
 			"ignore-as-root": false,
 			"web-dir": ".Build/Web",
 			"app-dir": ".Build"
+		},
+		"branch-alias": {
+			"dev-main": "v3.x-dev"
```

Comment:
This should be:

```composer
"dev-v3.0x": "v3.x-dev"
```

Personally, I find the "v3.0x" decision for the branch not that good. Would have favorized "3.0" or simply "3" for it.

Reply:
I change to 3.0, thanks :)